iothub/service/src/DigitalTwin/Models/DigitalTwinCommandResponse.cs

```diff
@@ -17,5 +17,10 @@ public class DigitalTwinCommandResponse
         /// Command invocation result payload, as supplied by the device.
         /// </summary>
         public string Payload { get; internal set; }
+
+        /// <summary>
+        /// Server Generated Request Id (GUID), to uniquely identify this request in the service.
```

Comment:
Unnecessary capitalization.

```suggestion
        /// Server generated request Id (GUID), to uniquely identify this request in the service.
```